web-dev/Frontend/Libraries & Frameworks/Learn-React.md

```diff
@@ -371,3 +371,7 @@ React is an open source javaScript library for building user interfaces by Faceb
 | Concept       |                                                 Best Video Resource                                                 |                                         Best Text Resource                                         |                             Documentation                             | Duration |
 | :------------ | :-----------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------: | :------: |
 | Reconcilation | [Chariot Solutions](https://chariotsolutions.com/screencast/philly-ete-2016-30-jim-sproch-react-js-reconciliation/) | [Medium](https://medium.com/@gethylgeorge/how-virtual-dom-and-diffing-works-in-react-6fc805f9f84e) | [ReactJS/Reconcilation](https://reactjs.org/docs/reconciliation.html) | 10 hours |
+
+# Courses
+
+- (The Beginner's Guide To React)[https://egghead.io/courses/the-beginner-s-guide-to-react]
```

Comment:
can you please check the typo here in rendering
![image](https://user-images.githubusercontent.com/17708702/41660554-ffbde22e-74b9-11e8-98ae-164c6247bc8e.png)

Reply:
@M-ZubairAhmed Good catch! I'll fix it!